docs/pages/protocol/signatures.mdx

```diff
@@ -4,11 +4,61 @@ description: "Learn about account signature types when using XMTP"
 
 # Account signatures with XMTP
 
-Learn about the types of account signatures you may be prompted to provide when using apps built with XMTP.
+Learn about the types of account signatures you might be prompted to provide when using apps built with XMTP.
 
-## Sign to send and receive messages using apps built with XMTP
+## Sign to create an inbox (first-time use)
 
-The first time you use an app built with XMTP (Extensible Message Transport Protocol) to send and receive messages using a blockchain account (account), you’re prompted to provide two signatures using your account keys:
+The first time you use an app built with XMTP, a **Sign this message?** window displays to request that you sign an **XMTP : Authenticate to inbox** message. Specifically, the message requests that you sign a **Create inbox** message with your wallet account. 
+
+Sign the message with your wallet account to consent to creating an XMTP inbox owned by your wallet account.
+
+<img width="400" src="https://raw.githubusercontent.com/xmtp/docs-xmtp-org/refs/heads/main/docs/pages/img/authen-to-inbox.PNG" alt="MetaMask wallet browser extension Sign this message? window showing an XMTP: Authenticate to inbox message" />
+
+## Sign to add another account to your inbox
```

Comment:
Just confirming, are we settling on the term 'account' for wallet-level identity, and 'inbox' for XID-level identity? Or should we just use the term 'wallet' rather than 'account'?

Reply:
@richardhuaaa Thank you for this question - I'm so grateful. I would love to go with wallet -- and just to be sure I understand - wallet as shorthand for "wallet address," is that right? =)

I used "account" based on an earlier pattern we set that called these signatures - "account signatures." But I think my correlation here is incorrect.

For example, to link a wallet address to an inbox, am I signing with my wallet account or wallet address?